pkg/cli/internal/initialize/init.go

```diff
@@ -146,9 +146,9 @@ func makeInitCmd() *cobra.Command {
 	var processors []string
 	cmd := &cobra.Command{
 		Use:   "init",
-		Short: "initializes a Conduit data directory",
-		Long: `Initializes a conduit.yml file and writes it to stdout. By default
-the config file uses an algod importer in follower mode and a block
+		Short: "Initializes a conduit configuration",
```

Comment:
Changing the description in line with the other commands. If you try `conduit help` you'll see what I'm talking about.